crucible-mir/CHANGELOG.md

```diff
@@ -1,7 +1,16 @@
-# next -- TBA
+# 0.9 -- 2024-08-30
 
-* Fix a bug in which `crucible-mir` would fail to parse MIR JSON code involving
-  casts from array references to pointers.
+* Implement byte-to-char casts for crucible-mir
+
+* Fix a bug in which `crucible-mir` would fail to parse MIR JSON code involving casts from array references to pointers.
+
+* Rearranged handling of constant slices into a reference and a separate static allocation for the body the reference points to.
+
+* Add support for GHC 9.8
+
+* Properly parse ArrayToPointer casts
```

Comment:
A minor thing, but the convention up until this point has been not to put lines of whitespace between each bullet point. Let's stick with that convention.